yt_dlp/aes.py

```diff
@@ -43,6 +66,10 @@ def aes_cbc_decrypt(data, key, iv):
     @param {int[]} iv          16-Byte IV
     @returns {int[]}           decrypted data
     """
+    if AES:
+        data, key, iv = map(intlist_to_bytes, (data, key, iv))
```

Comment:
we are converting bytes to intlist back to bytes for no reason :/
same when returning

Reply:
I'm aware. I'll look into it.

Reply:
Changing signature should be done consistently over the module, which is just a little bit of busy work.
My concern is, that `aes_encrypt` and `aes_ctr_decrypt` functions, are used both internally and externally, resulting the issue to be only kicked around.